libs/input-mapping/src/Helper/FakeDevStorageTableRewriteHelper.php

```diff
@@ -52,14 +53,14 @@ private static function getNewSource(string $source, string $branchName): string
             $bucketId = substr($bucketId, 2);
             $prefix = 'c-';
         }
-        $bucketId = $branchName . '-' . $bucketId;
+        $bucketId = $branchId . '-' . $bucketId;
```

Comment:
Je to tady opravdu branchId? Neni to branchName? 

```
$clientWrapper->getBranchClientIfAvailable()->webalizeDisplayName(
                (string) $clientWrapper->getBranchId()
            )['displayName']

Tohle spis vypada, ze vraci name? :)

Reply:
Ježišikriste teď jsem se fakt vyděsil, ne naštěstí ne - to `displayName` je field, kterej vrací ten webalize endpoint. Dovnitř vstupuje branchId(), vůbec nevím proč tu ten webalize je, ale teď to nemíním odstraňovat

Reply:
Chapu, promin, pusobilo to divne, hlavne, kdyz tam predtim bylo to branchName, ale byla by picovina, kdyby v id bucketu bylo "branch name", ale radsi jsem se zeptal :)

Reply:
No právě už jsem se jednou vyděsil včera - koukám na to - šmarja co tam dělá branchName, aha to je jen blbě pojmenovaná proměnná, no a teď znova to samý :)

Reply:
Ale odeslal, akorát do vedlejšího PR https://github.com/keboola/platform-libraries/pull/126#pullrequestreview-1528554066, tohle bylo původně basnutý nad ním